src/mme-app/handlers/attach_initue.c

```diff
@@ -584,15 +596,50 @@ handle_imsi_list_req(struct monitor_imsi_req *mir, int sock_fd)
 
 	return;
 }
-
+#endif
+void
+handle_imsi_list_req(struct monitor_imsi_req *mir, int sock_fd)
```

Comment:
we should open 2 subtask
1. Handle the prometheus query if number of IMSI's are huge.
2. Keep active list of IMSIs so that we don't have to go through big loop.

Reply:
TBD. Will open task for it.